.github/CODEOWNERS

```
@@ -1,7 +1,10 @@
 # Each line is a file pattern followed by one or more owners.
 
-# These owners will be the default owners for everything in
-# the repo. Unless a later match takes precedence,
-# these owners will be requested for review when someone
-# opens a pull request.
-*       @bjb28
+# These owners will be the default owners for everything in the
+# repo. Unless a later match takes precedence, these owners will be
+# requested for review when someone opens a pull request.
+* @ameliav @BenBreaksThings @nickviola @JCantu248 @dav3r @felddy @hillaryj @jsf9k @mcdonnnj
+
+# These folks own any files in the .github directory at the root of
+# the repository and any of its subdirectories.
+/.github/ @ameliav @BenBreaksThings @nickviola @JCantu248 @dav3r @felddy @hillaryj @jsf9k @mcdonnnj
```

This is just a fallback so our team is pulled into reviews that interact with the CI pieces even if we're not CODEOWNERS for the repo otherwise.

```suggestion
/.github/ @dav3r @felddy @jsf9k @mcdonnnj
```

---

Corrected as part of https://github.com/cisagov/pca-report-library/pull/3/commits/550dc34b08768dd4ac0d4b38b83a1f6893d4333f

---

Hi @mcdonnnj this has been corrected/updated, can you resolve the conversation? (Commit shown in Dave's above comment) Thanks